contracts/liquidityMining/KyberFairLaunchV2.sol

```diff
@@ -194,9 +202,9 @@ contract KyberFairLaunchV2 is IKyberFairLaunchV2, PermissionAdmin, ReentrancyGua
     poolInfo[poolLength].lastRewardTime = _startTime;
     poolInfo[poolLength].vestingDuration = _vestingDuration;
 
-    for (uint256 i = 0; i < _rewardPerSeconds.length; i++) {
+    for (uint256 i = 0; i < _totalRewards.length; i++) {
       poolInfo[poolLength].poolRewardData[i] = PoolRewardData({
-        rewardPerSecond: _rewardPerSeconds[i],
+        rewardPerSecond: _totalRewards[i].mul(multipliers[i]).div(_endTime - _startTime),
```

Comment:
maybe add a require that the calculated value != 0. just in case total reward amount is too small.

```
uint256 _rewardPerSecond = _totalRewards[i].mul(multipliers[i]).div(_endTime - _startTime);
require(_rewardPerSecond != 0, 'reward too small');
rewardPerSecond = _rewardPerSecond;
```

Reply:
same for `updatePool()`

Reply:
updated, tks